### PR TITLE
Added collections of named colormaps, attribute access

### LINKED ISSUE #2: mpl and colorcet names
I noticed that both mpl and colorcet uses the name 'inferno' for two different colormaps. The cc inferno map looks very similar to the mpl plasma map.

https://bokeh.github.io/colorcet/
http://matplotlib.org/users/colormaps.html

### PR DESCRIPTION
**Pull request by jbednar**
Misc collected fixes/improvements:

- Renamed inferno to bmy to fix #2.
- Updated code and notebooks for Python3
- Added attribute access to dictionaries for convenience.
- Added dictionaries for the named subset of colormaps, for use in GUI apps.

Example of those last two:

image

### CODE REVIEW

**Comment by jbednar** [comment]
Also fixes #4.

File: assets/CET_to_py.py [hunk 1]
```diff
@@ -31,12 +34,25 @@
...
-__version__ = '0.9.0'
+__version__ = '0.9.1'
 
 from collections import OrderedDict
+
+class AttrODict(OrderedDict):
+    """Ordered dictionary with attribute access (e.g. for tab completion)"""
+    def __getattr__(self, name): return self[name]
+    def __delattr__(self, name): del self[name]
+    def __setattr__(self, name, value): self[name] = value
```
Comment (philippjfr) [inline assets/CET_to_py.py]: Should this also have a ``__dir__`` method for tab completion or would that be too cluttered:

```python
    def __dir__(self): return self.keys()
```

File: colorcet/__init__.py [hunk 2]
```diff
@@ -19,12 +24,26 @@
...
 
 __version__ = '0.9.1'
 
 from collections import OrderedDict
+
+class AttrODict(OrderedDict):
+    """Ordered dictionary with attribute access (e.g. for tab completion)"""
+    def __getattr__(self, name): return self[name]
+    def __delattr__(self, name): del self[name]
+    def __setattr__(self, name, value): self[name] = value
```
Comment (philippjfr) [inline colorcet/__init__.py]: Comment above probably applies more here, a ``__dir__`` might be nice.

Comment (jbednar) [reply colorcet/__init__.py]: The first one is the source code for the second one, so sure, I'll add that.  Thanks!

**Pull request merged**